### DISCUSSION TITLE
How to execute dos commands in react-native-windows?

### DISCUSSION THREAD

**Discussion by gangli66** [microsoft/react-native-windows#10500] [Q&A] [answered]
I want to use react native to execute dos commands. Can any package or method to implement it?

**Comment by chrisglein** [comment, marked as answer]
There's nothing I'm aware of in RNW that is going to help you directly with executing terminal commands (which is what I assume you mean by DOS commands). Your best bet is to find what you're looking for from the general UWP APIs, and then figure out how you want to access that from JavaScript. Start here: https://microsoft.github.io/react-native-windows/docs/native-code

**Reply by gangli66** [reply]
Thanks for your answer.^_^
